doc_src/license.hdr

```diff
@@ -9,7 +9,9 @@
 
 `fish` Copyright © 2005-2009 Axel Liljencrantz. `fish` is released under the GNU General Public License, version 2.
 
-`fish` includes other code licensed under the GNU General Public License, version 2, including GNU `printf`:
+`fish` includes other code licensed under the GNU General Public License, version 2, including GNU `printf`.
+
+`fish` includes code licensed under the MIT License, such as the MuParser math expression evaluation library.
```

Comment:
The block about using the MIT license needs to go with the MIT license; this section is listing code that uses the GPL. There should be no "such as" - we should describe exactly the code that uses this license.